els/frontend/src/utils/const.js

```diff
@@ -0,0 +1,7 @@
+const config = {
+  headers: {
+    'Content-Type': 'application/json',
+  },
+};
+
+export default config;
```

Comment:
By convention as this is a constant variable, rename the variable to uppercase. In the case of multiple words, separate it with an underscore (`_`) e.g `export const FOO_BAR = 1`
```suggestion
const CONFIG = {
  headers: {
    'Content-Type': 'application/json',
  },
};

export default CONFIG;
```